cmd/postpolicyform.go

```diff
@@ -51,6 +53,18 @@ var startsWithConds = map[string]bool{
 	"$x-amz-date":              false,
 }
 
+var postPolicyIgnoreKeys = map[string]bool{
+	"Policy":              true,
+	xhttp.AmzSignature:    true,
+	xhttp.ContentEncoding: true,
+	http.CanonicalHeaderKey(xhttp.AmzChecksumAlgo):   true,
+	http.CanonicalHeaderKey(xhttp.AmzChecksumCRC32):  true,
+	http.CanonicalHeaderKey(xhttp.AmzChecksumCRC32C): true,
+	http.CanonicalHeaderKey(xhttp.AmzChecksumSHA1):   true,
+	http.CanonicalHeaderKey(xhttp.AmzChecksumSHA256): true,
+	http.CanonicalHeaderKey(xhttp.AmzChecksumMode):   true,
```

Comment:
this must not be ignored @jiuker @klauspost I think.

Reply:
Yeah. Seems like creating a bug rather than fixing the test.